.github/workflows/reusable_test_packages.yaml

```diff
@@ -43,6 +43,14 @@ jobs:
           tar -xvf $(ls falco-*.tar.gz)
           cd falco-${{ inputs.version }}-${{ inputs.arch }}
           sudo cp -r * /
+      
+      # Note: most probably the plugin related tests should be moved to the plugin repo sooner or later.
+      - name: Install needed artifacts using falcoctl
+        if: ${{ inputs.static == false }}
+        run: |
+          sudo mkdir -p /usr/share/falco/plugins
```

Comment:
so the e2e tests now look at plugins + rules under `/usr/share/falco/plugins` instead of relying on the ones bundled in the `tar.gz`. This is possible without modification because `/usr/share/falco/` and `/etc/falco` are the default paths, isn't it?

Reply:
Yes